### PR TITLE
amq set up to listen for request from the identity service

### PR DESCRIPTION
**Pull request by SunmadeBabafemi**

### CODE REVIEW

File: src/listener.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+async function bootstrap() {
+  const app = await NestFactory.createMicroservice<MicroserviceOptions>(
+    AppModule,
+    {
+      transport: Transport.RMQ,
+      options: {
+        urls: [process.env.RABBITMQ_URL],
+        queue: process.env.RABBITMQ_QUEUE,
+        queueOptions: {
+          durable: false,
```
Comment (Chimexx) [inline src/listener.ts:17]: You need to change this to true, because the core service it is listening to is set to true, else it is throwing an error

**Review by Chimexx — CHANGES_REQUESTED** [review]

File: src/listener.ts (hunk 1 above)
Comment (SunmadeBabafemi) [reply src/listener.ts:17]: i set it to true before.. it threw error saying i should set it to false.. let me find a away around it

Comment (SunmadeBabafemi) [reply src/listener.ts:17]: Screenshot from 2022-08-29 14-58-47

Comment (Chimexx) [reply src/listener.ts:17]: Hmmm, it throws error on my side if set to false, true works

Comment (SunmadeBabafemi) [reply src/listener.ts:17]: wahala

**Review by Chimexx — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**